Dockerfile

```diff
@@ -16,4 +11,9 @@ ENTRYPOINT ["sh", "./flask.sh" ]
 FROM base as test
 ENV PATH = "${PATH}:/root/todo_app"
 ENTRYPOINT ["poetry", "run", "pytest"]
-CMD ["todo_app"]
\ No newline at end of file
+CMD ["todo_app"]
+
+FROM base as production
```

Comment:
Assuming you've reordered this because of the action limitation of not being able to specify a target stage, a quick comment here feels very worthwhile - it would be natural for a new dev to add a new stage after this and they might be very confused how adding a totally new unrelated stage breaks the production build!